src/server/node_manager.c

```diff
@@ -467,8 +467,9 @@ get_addr_of_nodebyname(char *name, unsigned int *port)
 
 
 enum Set_All_State_When {
-	Set_ALL_State_All_Down,	 /* set set on vnodes when all Moms are down */
-	Set_All_State_Regardless /* set set on vnodes regardless */
+	Set_ALL_State_All_Down,	  /* set set on vnodes when all Moms are down */
+	Set_All_State_Regardless, /* set set on vnodes regardless */
```

Comment:
set appears twice in the previous two lines

Reply:
I'll fix that